### DISCUSSION TITLE
Retina (2x) thumbnails not being resized correctly without transformations?

### DISCUSSION THREAD

**Discussion by leuchtdiode** [pimcore/pimcore#14440] [Q&A / Support]
Hi,

I'm using pimcore version 10.5.11 and experience a problem with image thumbnails. Retina images (2x) are not being "doubled in size" when there are no transformations set in the thumbnail config.

Look at these images:

This thumbnail has a transformation "cover (focal point support)" 400x250px and the images 1x and 2x are getting resized correctly:

https://car-t-cell.com/_Portraits/224/image-thumb__224__article-list/AdobeStock_279977730_k_e_n_400x.webp
https://car-t-cell.com/_Portraits/224/image-thumb__224__article-list/AdobeStock_279977730_k_e_n_400x@2x.webp

Whereas this thumbnail has no transformations and the image size stays the same (728x90 px) for 1x and 2x:

https://car-t-cell.com/Banner/225/image-thumb__225__advertisement/Banner_Car-T-Cell_728x90_b.webp
https://car-t-cell.com/Banner/225/image-thumb__225__advertisement/Banner_Car-T-Cell_728x90_b@2x.webp

Is this expected behaviour or a bug? Or how can I achieve correct resizing without having any transformation? I'm using a thumbnail without transformation to get the picture tag with .webp images.

Thx & BR,
Alex

**Comment by NiklasBr** [comment]
Which version of ImageMagick are you using? Could it be you are experiencing [this issue](https://github.com/ImageMagick/ImageMagick/issues/4931)?

**Reply by leuchtdiode** [reply]
Hi,

thanks for pointing that out. Actually I don't know what pimcore is exeucting with imagick to convert the image. But as I'm using 6.9.10 of Imagick, it could be that I'm at a faulty version regarding your issue.

I will analyze further.

Thx & BR,
Alex
